### PR TITLE
The circle of life.

### PR DESCRIPTION
**Pull request by aaronschachter**
Creates new user from legacy JSON object.

### CODE REVIEW

**Comment by angaither** [comment]
there's no simba picture here? reject.

File: lib/modules/dosomething/dosomething_user/dosomething_user.module [hunk 1]
```diff
@@ -445,6 +445,74 @@ function dosomething_user_create_user_by_mobile($number) {
...
+  if (isset($object['user']) && isset($object['profile'])) { 
+    $user = array();
+    // Will be overridden by uid.
+    $user['name'] = user_password(8);
+    $user['pass'] = $form_state['values']['pass'];
+    $user['mail'] = $object['user']['mail'];
+    $user['created'] = $object['user']['created'];
+    $user['access'] = $object['user']['access'];
+    // Set the init value as legacy uid.
+    $user['init']= $object['user']['uid'];
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_user/dosomething_user.module]: probably don't need this one anymore.

Comment (angaither) [inline lib/modules/dosomething/dosomething_user/dosomething_user.module]: you'll set `$user['uid'] = $object['user']['uid']`

**Comment by desmondmorris** [comment]
+1

**Pull request merged**